doc/api/thaliemitter.md

```diff
@@ -0,0 +1,189 @@
+# The `ThaliEmitter` class
+
+The `ThaliEmitter` class is a bridge between the native Cordova plugin and JavaScript to give you an easy to use interface for broadcasting availability and to connect and disconnect from peer clients.
+
+## Usage
+
+This is the basic usage to listen for the `peerAvailabilityChanged` event as well as start broadcasting the availability of the current device.
```

Comment:
This is the basic usage that in order to actually do something useful, you need to listen to the event before doing anything.  This is the overall usage of the class, not just the specific APIs